integration-tests/tests/analysis-context-bar.js

```diff
@@ -62,5 +62,6 @@ const testAnalysisContextBarFn = _.flow(
 registerTest({
   name: 'analysis-context-bar',
   fn: testAnalysisContextBarFn,
-  timeout: 15 * 60 * 1000
+  timeout: 15 * 60 * 1000,
+  targetEnvironments: ['alpha', 'perf', 'staging']
```

Comment:
The tests still run on the PR if 'local' is in the array, which isn't what I would have expected. Omitting both 'local' and 'dev'.

Reply:
We never actually run the tests with `environment` set to dev.

The `integration-tests-against-dev` job sets the `env` parameter to `local`.
https://github.com/DataBiosphere/terra-ui/blob/9045a42b031d4c3426f6fcd9ef0f6bb4f776fee9/.circleci/config.yml#L243-L249

Which gets passed to integration test code via the `ENVIRONMENT` environment variable.
https://github.com/DataBiosphere/terra-ui/blob/9045a42b031d4c3426f6fcd9ef0f6bb4f776fee9/.circleci/config.yml#L154-L165

And that's used to determine whether or not to run the test.
https://github.com/DataBiosphere/terra-ui/blob/9045a42b031d4c3426f6fcd9ef0f6bb4f776fee9/integration-tests/utils/jest-utils.js#L12
https://github.com/DataBiosphere/terra-ui/blob/9045a42b031d4c3426f6fcd9ef0f6bb4f776fee9/integration-tests/utils/jest-utils.js#L29